docs/source/operations/indy_networkyaml.md

```diff
@@ -0,0 +1,277 @@
+        branch: "gitops_branch"                     # Git branch where release is being made
+        release_dir: "gitops_release_dir"           # Relative Path in the Git repo for flux sync per environment. 
+        chart_source: "gitops_charts"               # Relative Path where the Helm charts are stored in Git repo
+        git_push_url: "gitops_push_url"             # Gitops https URL for git push like "github.com/hyperledger-labs/blockchain-automation-framework.git"
+        username: "git_username"                    # Git Service user who has rights to check-in in all branches
+        password: "git_password"                    # Git Server user password
+        email: "git_email"                          # Email to use in git config
+        private_key: "path_to_private_key"          # Path to private key file which has write-access to the git repo
+
+```
+
+The `gitops` field under each organization contains
+
+| Field       | Description                                              |
+|-------------|----------------------------------------------------------|
+| git_ssh                              | SSH url of the repository where flux should be synced                                                            |
+| branch                               | Branch of the repository where the Helm Charts and value files are stored                                        |
+| release_dir                          | Relative path where flux should sync files                                                                       |
+| chart_source                         | Relative path where the helm charts are stored                                                                   |
+| git_push_url                         | Gitops https URL for git push like "github.com/hyperledger-labs/blockchain-automation-framework.git"             |
+| username                             | Username which has access rights to read/write on repository                                                     |
+| password                             | Password of the user which has access rights to read/write on repository                                         |
+| email                                | Email of the user to be used in git config                                                                       |
+| private_key                          | Path to the private key file which has write-access to the git repo                                              |
+
+
+The services field for each organization under `organizations` section of Indy contains list of `services` which could be trustee/steward/endorser
+
+The snapshot of trustee service with example values is below
+```yaml
+      services:
+        trustees:
+        - trustee:
+          name: provider-trustee
+          genesis: true
+```
+
+The fields under `trustee` service are 
```

Comment:
Add a link to the definitions of trustee/steward/endorser from Indy documenatation